### ISSUE TITLE
Floating windows with borders doesn't work well with smooth scrolling

### ISSUE THREAD

**Issue by archiif** [neovide/neovide#1446]
**Describe the bug**
When smooth scrolling is activated with `--multigrid`, scrolling in a floating window with borders causes the border to shake, making it really dizzying and hard to follow.

**To Reproduce**
1. Launch neovide with `--multigrid`
2. Spawn a floating window with borders
3. Scroll in it

**Expected behavior**
The border in the floating window should not shake when scrolling

**Screenshots**
![2022-08-03_08-25-21](https://user-images.githubusercontent.com/60074051/182504416-6b7193af-cbe7-4bab-b2de-16e61c729d19.gif)

**Desktop (please complete the following information):**
 - OS: Microsoft Windows [Version 10.0.19043.1826]
 - Neovide Version 0.9.0
 - Neovim Version v0.8.0-dev-739-ge59bc078d

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**
[neovide_rCURRENT.log](https://github.com/neovide/neovide/files/9247226/neovide_rCURRENT.log)

**Additional context**

**Comment by AlexvZyl** [comment]
Yeah, I would love to try and fix it.  If one of the maintainers can point me in the right direction I would love to give it a shot (assuming that it is not super difficult...).
